airflow/ui/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc -b && vite build",
-    "lint": "eslint . && tsc --p tsconfig.app.json",
+    "lint": "eslint --quiet && tsc --p tsconfig.app.json",
```

Comment:
`--quite` makes the logs skip warnings, so they only show up in our editors.

Reply:
Why do we want to suppress warnings in our editors ?

The fix step bellow `lint:fix` does not specify the `--quiet` does that mean that we actually check `warnings` for errors in the CI but our editor is omitting them ?

Reply:
Oh, maybe I didn't explained it well, it's the other way around:

1. The warnings only show up in the editor, they don't show up in the CLI, because warnings are warnings, not errors.
2. When you run `fix` it means you're trying to well ... fix your linting issues, so there is fine to include the warnings.

Reply:
Oh yup! The plan with the warnings is to fix them when we see them in our editors, and then once that warning is no longer present in the code, just change that `WARN` to `ERROR`. So basically a "gradual" improvement. I have a personal preference to do the same and just make pretty much everything an error unless is something that can be a valid exception every now and then (for example `no-console` which should be fixed pretty much everywhere, unless you're writing actual CLI output for Node).